contracts/exchange/contracts/src/MixinStakingManager.sol

```diff
@@ -0,0 +1,67 @@
+/*
+
+  Copyright 2019 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.5.9;
+
+import "@0x/contracts-utils/contracts/src/Ownable.sol";
+import "./interfaces/IStakingManager.sol";
+
+
+contract MixinStakingManager is
```

Comment:
Thoughts on making the language in this contract more generic (e.g `MixinProtocolFeeManager`? The logic in the `Exchange` contract is pretty much entirely independent of the `Staking` contract.